src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -11,6 +11,7 @@
 #include "RequestReceiveContentArgs.g.cpp"
 
 #include <filesystem>
+#include <winrt/Microsoft.Management.Deployment.h>
```

Comment:
if you've got the WinRT header as generated from the winmd, then I would _think_ you wouldn't need to play silly CLSID games yourself...